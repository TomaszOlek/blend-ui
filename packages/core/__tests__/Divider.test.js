import React from "react";
import renderer from "react-test-renderer";
import Divider from "../src/Divider";
import ThemeProvider from "../src/theme/ThemeProvider";

test("Divider renders correctly", () => {
  const tree = renderer
    .create(
      <ThemeProvider>
        <Divider />
      </ThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
