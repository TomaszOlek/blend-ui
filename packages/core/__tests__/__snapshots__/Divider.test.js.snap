// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1w52cfi-0 fMEevm"
>
  <hr
    className="Divider__DividerElement-sc-qfr82c-0 bZUdke"
  />
</div>
`;
